pkg/upload/unity-android.go

```diff
@@ -37,53 +36,53 @@ func ProcessUnityAndroid(apiKey string, aabPath string, applicationId string, ve
 			}
 
 			if aabPath == "" {
-				aabPath, err = utils.FindLatestFileWithSuffix(path, ".aab")
-
-				if err != nil {
-					return err
-				}
+				aabPath, _ = utils.FindLatestFileWithSuffix(path, ".aab")
 			}
 		} else if strings.HasSuffix(path, ".symbols.zip") {
 			zipPath = path
 
 			if aabPath == "" {
 				buildDirectory := filepath.Dir(path)
 
-				aabPath, err = utils.FindLatestFileWithSuffix(buildDirectory, ".aab")
-
-				if err != nil {
-					return err
-				}
+				aabPath, _ = utils.FindLatestFileWithSuffix(buildDirectory, ".aab")
 			}
 		} else {
 			return fmt.Errorf(path + " is not a .symbols.zip file or containing directory")
 		}
 	}
 
-	log.Info("Extracting " + filepath.Base(aabPath) + " into a temporary directory")
+	if utils.FileExists(aabPath) {
```

Comment:
I think if no `aabPath` is provided then we should just silently skip AAB processing (i.e. you can remove the `else` branch of this). However if `aabPath` is provided and it doesn't exist, then we should error out as they've given an invalid input.